attribute.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+# Represents a classroom.
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
+    student.classroom = self
+  end
+end
+
+# Represents a Student.
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
+
+# Represents a Book.
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
+
+# Represents a Rental.
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] After initializing `book` and `person` instance variables in the `Rental` class, I'd suggest adding the rental instance itself to the `rentals` property of both the book and person objects achieving the 'belongs-to' association as shown below 👇
  ```rb
  book.rentals << self
  person.rentals << self
  ```